app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @merchant.name %></h1>
+
+<section class="links">
+  <%= link_to "Items", "items" %> <br>
+  <%= link_to "Invoices", "invoices" %>
+</section>
```

Comment:
looking at the wireframe, it looks like the links and merchants name are both part of the header. I'm thinking 
 we should wrap the entire thing inside a `<header></header>` tag so that if we get to styling it's better compartmentalized.